sumo-api/sumo/metrics/evaluator.py

```diff
@@ -291,12 +292,19 @@ def shape_score(self):
             # det_matches = 1 if correct detection, 0 if false positive
             det_matches = []
```

Comment:
Clearly should be a list of tuples generated by a list comprehension, but I won't nitpick too much...

Reply:
True, but the code is more readable with it written out.  Plus, the 80 column limit on code makes long, complicated list comprehensions even harder to understand.